app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -293,9 +293,7 @@ open class FenixApplication : LocaleAwareApplication(), Provider {
     }
 
     private fun startMetricsIfEnabled() {
-        if (settings().isTelemetryEnabled) {
-            components.analytics.metrics.start(MetricServiceType.Data)
-        }
+        components.analytics.metrics.start(MetricServiceType.Data)
```

Comment:
This is not intentional correct?

Reply:
Sorry, about that I missed that one,  I added it when I was testing locally